addon/components/o-s-s/layout/navbar/nav-item.hbs

```diff
@@ -0,0 +1,5 @@
+<div class={{concat 'fx-row fx-gap-px-6 fx-xalign-center border-radius-md oss-navbar-item' (if @active '--active' '')}}>
```

Comment:
Nit:
```hbs
<div class="oss-navbar-item {{if @active "oss-navbar-item--active"}}" >
```
```css
.oss-navbar-item {
  .border-radius-md;
  display: flex;
  flex-direction: row;
  gap: var(--spacing-px-6);
  align-items: center;
  ...
}
```
 Is something like the above snippet more readable ?

Reply:
yes !
Also I don't see any reason to use concat here in the first place 🤔

Reply:
Might want to put `fx-1` in there so that it uses the fullwidth of the parent.

I might regret this later 😅